test/cypress/features/transaction.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Transactions 
+
+  Background:
+    Given I login as testnet_guy on testnet
+    And I wait 5 seconds
+    Given I am on Transactions page
```

Comment:
If the step definition in `common.js` is used then this feature step can be done as

`Given I am on transactions page`

Reply:
@eniolam1000752 Fixed